lithops/standalone/proxy.py

```diff
@@ -130,10 +170,58 @@ def preinstalls():
     return response
 
 
+def install_evironemnt():
+    """
+    Install docker command and dependenices in case they are not installed
+    Only for Ubuntu-based OS
+    """
+
+    os_version = sp.check_output('uname -a', shell=True).decode()
+
+    if 'Ubuntu' in os_version:
+        try:
+            sp.check_output('docker ps > /dev/null 2>&1', shell=True)
+            docker_installed = True
+            logger.info("Environment already installed")
+        except Exception:
+            logger.info("Environment not installed")
+            docker_installed = False
+
+        if not docker_installed:
+            # If docker is not installed, nothing is installed, so lets install anything here
+            cmd = 'apt-get remove docker docker-engine docker.io containerd runc -y; '
+            cmd += 'apt-get update '
+            cmd += '&& apt-get install apt-transport-https ca-certificates curl gnupg-agent software-properties-common -y '
+            cmd += '&& curl -fsSL https://download.docker.com/linux/ubuntu/gpg | apt-key add - > /dev/null 2>&1 '
+            cmd += '&& add-apt-repository "deb [arch=amd64] https://download.docker.com/linux/ubuntu $(lsb_release -cs) stable" '
+            cmd += '&& apt-get update '
+            cmd += '&& apt-get install docker-ce docker-ce-cli containerd.io -y '
+            try:
+                logger.info("Installing Docker...")
+                with open(log_file, 'a') as lf:
+                    sp.run(cmd, shell=True, stdout=lf, stderr=lf, universal_newlines=True)
+                logger.info("Docker installed successfully")
+            except Exception as e:
+                logger.info("There was an error installing Docker: {}".format(e))
+
+            cmd = 'pip3 install -U lithops flask gevent && pip3 uninstall lithops -y'
```

Comment:
1. is there as case when vm has docker client installed already while missing required python packages?
2. why is there a need to install and uninstall lithops?

Reply:
1. If a user manually installed docker Yes, I think here we need to check if pip3 is installed, and if not run this command
2.This is a short way to install all lithops dependencies. I am not sure if it is necessary to uninstall lithops at the end, by put it just in case. need to test it, but in any case it doesn't produce any error.